client-templates/container-registry-agent/accept.json.sample

```diff
@@ -1,62 +1,67 @@
 {
   "public": [
-  {
-    "//": "image scan (discover) callback (v1)",
-    "method": "POST",
-    "path": "/api/v1/import/app"
-  },
-  {
-    "//": "image scan done callback (v2)",
-    "method": "POST",
-    "path": "/api/v2/import/done"
-  },
-  {
-    "//": "image scan result callback (v2)",
-    "method": "POST",
-    "path": "/api/v2/import/result"
-  }
+    {
+      "//": "image scan (discover) callback (v1)",
+      "method": "POST",
+      "path": "/api/v1/import/app"
+    },
+    {
+      "//": "image scan done callback (v2)",
+      "method": "POST",
+      "path": "/api/v2/import/done"
+    },
+    {
+      "//": "image scan (discover) callback (v1)",
+      "method": "POST",
+      "path": "/api/v1/recurring-tests/dependencies"
+    },
```

Comment:
Technically this is the only diff in this PR.
@ahmed-agabani-snyk, could we remove other formatting changes?

Reply:
I'll split the formatting and adding a new rule into two different PRs

Reply:
PR superseded by https://github.com/snyk/broker/pull/588

Reply:
PR superseded by https://github.com/snyk/broker/pull/589